whipper/command/mblookup.py

```diff
@@ -0,0 +1,44 @@
+from whipper.command.basecommand import BaseCommand
+from whipper.common.mbngs import musicbrainz
+
+
+class MBLookup(BaseCommand):
+    summary = "lookup MusicBrainz entry"
+    description = """Look up a MusicBrainz disc id and output information.
+
+You can get the MusicBrainz disc id with whipper cd info.
+
+Example disc id: KnpGsLhvH.lPrNc1PBL21lb9Bg4-"""
+
+    def add_arguments(self):
+        self.parser.add_argument(
+            'mbdiscid', action='store', help="MB disc id to look up"
+        )
+
+    def do(self):
+        try:
+            discId = unicode(self.options.mbdiscid)
+        except IndexError:
+            print 'Please specify a MusicBrainz disc id.'
+            return 3
```

Comment:
The return value of 3 has no specific meaning, right? (just returning non-zero)

Reply:
It may be semantic, but I haven't checked if it means anything by a particular standard. It's left over from morituri.